ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py

```diff
@@ -127,6 +138,15 @@ def collect(self, config):
         log.info('Start Collection of Events')
 
         ds0 = config.datasources[0]
+
+        try:
```

Comment:
What's the purpose of handling this exception you're generating here anyway? Wouldn't the following achieve the same thing?

```python
if ds0.params['query_error']:
    raise EventLogException('Please verify EventQuery on datasource %s'
        % ds0.params['eventid'])
```